synapseutils/migrate_functions.py

```diff
@@ -608,9 +633,18 @@ def migrate_indexed_files(
                     from migrations
                     where
                         status = ?
-                        and ((id > ? and type in (?, ?))
-                            or (id = ? and type = ? and version is not null and version > ?)
-                            or (id = ? and type = ? and (row_id > ? or (row_id = ? and col_id > ?))))
+                        and (
+                                (
+                                    ((id > ? and type in (?, ?))
+                                    or (id = ? and type = ? and version is not null and version > ?)
+                                    or (id = ? and type = ? and (row_id > ? or (row_id = ? and col_id > ?))))
```

Comment:
inline the static values such as "type" so it is easier to see which case is being handled

Reply:
Used a mix of named args and inline as above.